composer.json

```diff
@@ -6,9 +6,16 @@
     "require-dev": {
         "wp-coding-standards/wpcs": "^0.14.1",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.4.4",
+        "squizlabs/php_codesniffer": "^3.2",
```

Comment:
I'm not sure this is needed since `wp-coding-standards/wpcs` requires it in its dependencies. Only if you want to enforce the latest version separately.

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/composer.json

Reply:
Let's simplify as much as possible. @hellofromtonya can you take this line out (or provide a reason why it needs to stay)?

Reply:
I agree with @dingo-d.  It's already a dependency in WPCS and not necessary to be in our composer file.  Will remove.